marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/builtins/steps/mapping/entity-services/lib.sjs

```diff
@@ -115,6 +115,20 @@ function fetchNamespacesFromMappingStep(mappingStep){
   return namespaces;
 }
 
+function fetchMappingNamespacesObject(mappingStep){
```

Comment:
Let's use "get", which implies we're getting something from an object and not doing a query. "fetch" isn't used much elsewhere and thus raises the question of "What does fetch mean here?", which could imply a query.